host-spec/c06-consensus.tm

```diff
@@ -241,6 +296,20 @@
 
   <subsection|Block Production Lottery>
 
+  <\definition>
+    <label|defn-babe-constant>The <with|font-series|bold|BABE constant>
+    <math|<with|font-series|medium|c>\<in\><around*|(|0,1<rsub|>|)>> is the
+    probability that a slot will be not be empty. It is initialized at
```

Comment:
is the probability that a slot will be not be empty -> is the probability that a slot will not be empty